test/integration/master/api/providers_controller_integration_test.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class Master::Api::ProvidersControllerIntegrationTest < ActionDispatch::IntegrationTest
 
   def setup
     @account_plan = master_account.default_account_plan
-    @account_plan.update_attribute(:approval_required, false)
+    @account_plan.update(approval_required: false)
```

Comment:
Given that DB should be truncated after test, this should be a good approach. I would suggest to use the `!` form though.

```suggestion
    @account_plan.update!(approval_required: false)
```